Makefile

```
@@ -12,6 +12,10 @@ TAG=$(shell cat CI_TAG)
 endif
 endif
 
+ifeq ($(findstring x,$(TAG)),x)
+QUAY_TAG_EXPIRATION="--label quay.expires-after=1w"
```

I think this cause an error. Should we drop `"`
```suggestion
QUAY_TAG_EXPIRATION=--label quay.expires-after=1w
```